miniwallet/config.ts

```diff
@@ -0,0 +1,146 @@
+import { ethers } from 'ethers'
+import 'dotenv/config'
+
+export default {
+  mainnet: {
+    miniWallet: {
+      initialOperatorThreshold: process.env.MINIWALLET_INITIAL_OPERATOR_THRESHOLD,
+      initialOperators: JSON.parse(process.env.MINIWALLET_INITIAL_OPERATORS || '[]'),
+      initialUserLimit: ethers.utils.parseEther(process.env.MINIWALLET_INIITIAL_USER_LIMIT || '1000000'),
+      initialAuthLimit: ethers.utils.parseEther(process.env.MINIWALLET_INIITIAL_AUTH_LIMIT || '100000')
+    },
+    miniID: {
+      baseUri: process.env.MINIID_BASE_URI || 'ipfs://Qmc8DVEthq7cZMTMyZ2NQ8dHkG99n549DMBwNzAypQgXe1/MiniID'
+    },
+    mini721: {
+      saleIsActive: process.env.MINI721_DEPLOY_SALES_IS_ACTIVE || false,
```

Comment:
process.env always returns string. Each config variable should have a fixed type

Reply:
Thanks 🙏 
updated with
`    saleIsActive: process.env.MINI721_DEPLOY_SALES_IS_ACTIVE === '1' || process.env.MINI721_DEPLOY_SALES_IS_ACTIVE === 'true',`